webapp/src/templates/partials/reports_content.html

```diff
@@ -35,8 +35,10 @@
           </div>
 
           <div ng-if="summary.subject.type" class="subject">
-            <span class="name"
+              <a ng-href="#/contacts/{{summary.subject._id || summary.fields.patient_uuid}}">
+                <span class="name"
                   ng-bind="(summary.subject.type === 'name' && summary.subject.value) || summary.subject.name || ('report.subject.unknown' | translate)"></span>
+              </a>
```

Comment:
I think it's possible to have no `summary.subject._id` or `summary.fields.patient_uuid`. In this case we'd want to not have the link. So I'm thinking we should add a `ng-if` to revert to the old behaviour if the ID isn't known. What do you think?

Reply:
Yeah I think that is right, so we don't end up with broken links